doc/user_guide/user_guide.md

```diff
@@ -216,6 +216,25 @@ Accordingly the creation of an adapter script looks like this:
 final AdapterScript adapterScript = schema.createAdapterScript("HELLO_WORLD", "PYTHON", "print \"Hello World\"");
 ```
 
+#### Debugging (Java only)
+
+Exasol Java aapter scripts support remote debugging (see [remote debugging in virtual schemas](https://github.com/exasol/virtual-schemas/blob/master/doc/development/remote_debugging.md)).
+This requires a special option for the `CREATE ADAPTER SCRIPT` command. The TDDB can also add this option.
```

Comment:
```suggestion
This requires a special option for the `CREATE ADAPTER SCRIPT` command. The TDDB can also add this option. When this option is set, the Java environment the script runs in uses a debug agent that can connect to a remote debugger.
```

Reply:
```suggestion
This requires a special option for the `CREATE ADAPTER SCRIPT` command. The TDDB can also add this option. When this option is set, the Java environment the script runs in uses a debug agent that connects to a remote debugger.
```

Reply:
I wanted to make clear that it is not optional. When the parameter is set, the VM crashes if no debugger is avaiable.

Reply:
OK.